level2/derived3.lisp

```diff
@@ -179,3 +186,13 @@ or otherwise it can be anything (e.g. (take-while '(a . b) (constantly t)) retur
 (defpattern λlist (&rest patterns)
   "Alias to lambda-list"
   (compile-destructuring-pattern (parse-lambda-list patterns)))
+
+(defpattern lambda-list-nc (&rest patterns)
+  "lambda-list but with additional non-canonical syntax,
+   1. (lambda-list &key a &allow-other-keys rem)
+      REM collects every key that wasn't declared"
```

Comment:
please add note about the difference to &rest. &rest args collects all elements.